operations/CSM_product_management/Run_NCN_Personalization.md

```diff
@@ -0,0 +1,167 @@
+# Run NCN Personalization
+
+NCN personalization is the process of applying product-specific configuration
+to NCN nodes post-boot. Prior to running this procedure, gather the following
+information required by [CFS](../configuration_management/Configuration_Management.md)
+to create a [configuration layer](../configuration_management/Configuration_Layers.md):
+
+* HTTP clone URL for the configuration repository in [VCS](../configuration_management/Version_Control_Service_VCS.md),
+* Path to the Ansible play to run in the repository,
+* Commit ID in the repository for CFS to pull and run on the nodes.
+
+Products may supply multiple plays to run, in which case multiple configuration
+layers must be created.
```

Comment:
This might be a good place to refer to the Getting Started Guide for a list of all products that perform NCN Personalization.

Reply:
Fixed